tests/test_autoint.py

```diff
@@ -142,52 +142,84 @@ def test_classification(
         pred_df = tabular_model.predict(test)
         assert pred_df.shape[0] == test.shape[0]
 
-# import numpy as np
-# import pandas as pd
-# from sklearn.datasets import fetch_california_housing, fetch_covtype
-
-
-# def regression_data():
-#     dataset = fetch_california_housing(data_home="data", as_frame=True)
-#     df = dataset.frame.sample(5000)
-#     df["HouseAgeBin"] = pd.qcut(df["HouseAge"], q=4)
-#     df["HouseAgeBin"] = "age_" + df.HouseAgeBin.cat.codes.astype(str)
-#     test_idx = df.sample(int(0.2 * len(df)), random_state=42).index
-#     test = df[df.index.isin(test_idx)]
-#     train = df[~df.index.isin(test_idx)]
-#     return (train, test, dataset.target_names)
-
-
-# def classification_data():
-#     dataset = fetch_covtype(data_home="data")
-#     data = np.hstack([dataset.data, dataset.target.reshape(-1, 1)])[:10000, :]
-#     col_names = [f"feature_{i}" for i in range(data.shape[-1])]
-#     col_names[-1] = "target"
-#     data = pd.DataFrame(data, columns=col_names)
-#     data["feature_0_cat"] = pd.qcut(data["feature_0"], q=4)
-#     data["feature_0_cat"] = "feature_0_" + data.feature_0_cat.cat.codes.astype(str)
-#     test_idx = data.sample(int(0.2 * len(data)), random_state=42).index
-#     test = data[data.index.isin(test_idx)]
-#     train = data[~data.index.isin(test_idx)]
-#     return (train, test, ["target"])
 
+@pytest.mark.parametrize("multi_target", [True, False])
+@pytest.mark.parametrize(
+    "continuous_cols",
+    [
+        [
+            "AveRooms",
+            "AveBedrms",
+            "Population",
+            "AveOccup",
+            "Latitude",
+            "Longitude",
+        ],
+    ],
+)
+@pytest.mark.parametrize("categorical_cols", [["HouseAgeBin"]])
+@pytest.mark.parametrize("continuous_feature_transform", [None])
+@pytest.mark.parametrize("normalize_continuous_features", [True])
+@pytest.mark.parametrize("target_range", [True, False])
```

Comment:
We can reduce the parameters a bit here... for eg. target_range will have no impact on the ssl task. Same feedback in all the other ssl test cases as well

Reply:
I have removed some of the test cases that have no impact on ssl